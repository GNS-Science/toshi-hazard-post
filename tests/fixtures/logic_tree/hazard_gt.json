{
  "data": {
    "node1": {
      "id": "R2VuZXJhbFRhc2s6NjcwMTI1NA==",
      "children": {
        "total_count": 3,
        "edges": [
          {
            "node": {
              "child": {
                "arguments": [
                  {
                    "k": "title",
                    "v": "OpenQuake Hazard Calcs"
                  },
                  {
                    "k": "description",
                    "v": "Logic Tree 9.0.1, locations for cave locations"
                  },
                  {
                    "k": "task_type",
                    "v": "HAZARD"
                  },
                  {
                    "k": "gmcm_logic_tree",
                    "v": "<?xml version=``1.0`` encoding=``UTF-8``?>--<nrml xmlns:gml=``http://www.opengis.net/gml``-      xmlns=``http://openquake.org/xmlns/nrml/0.4``>-    <logicTree logicTreeID='lt1'>-            <logicTreeBranchSet uncertaintyType=``gmpeModel`` branchSetID=``bs_crust``-                    applyToTectonicRegionType=``Active Shallow Crust``>--                    <logicTreeBranch branchID=``BSSA2014_center``>-                <uncertaintyModel>[BooreEtAl2014]-                  sigma_mu_epsilon = 0.0 </uncertaintyModel>-                        <uncertaintyWeight>1.0</uncertaintyWeight>-                    </logicTreeBranch>--            </logicTreeBranchSet>--            <logicTreeBranchSet uncertaintyType=``gmpeModel`` branchSetID=``bs_interface``-                    applyToTectonicRegionType=``Subduction Interface``>--                <logicTreeBranch branchID=``ATK22_SI_center``>-                    <uncertaintyModel>[Atkinson2022SInter]-                          epistemic = ``Central``-                            modified_sigma = ``true``-                           </uncertaintyModel>-                    <uncertaintyWeight>1.0</uncertaintyWeight>-                      </logicTreeBranch>--            </logicTreeBranchSet>--            <logicTreeBranchSet uncertaintyType=``gmpeModel`` branchSetID=``bs_slab``-                    applyToTectonicRegionType=``Subduction Intraslab``>--                <logicTreeBranch branchID=``ATK22_SS_center``>-                      <uncertaintyModel>[Atkinson2022SSlab]-                              epistemic = ``Central``-                              modified_sigma = ``true``-                              </uncertaintyModel>-                        <uncertaintyWeight>1.0</uncertaintyWeight>-                </logicTreeBranch>--            </logicTreeBranchSet>-    </logicTree>-</nrml>-"
                  },
                  {
                    "k": "model_type",
                    "v": "COMPOSITE"
                  },
                  {
                    "k": "intensity_spec",
                    "v": "{'tag': 'fixed', 'measures': ['PGA'], 'levels': [0.01, 0.02, 0.04, 0.06, 0.08, 0.1, 0.2, 0.3, 0.4, 0.5, 0.6, 0.7, 0.8, 0.9, 1.0, 1.2, 1.4, 1.6, 1.8, 2.0, 2.2, 2.4, 2.6, 2.8, 3.0, 3.5, 4.0, 4.5, 5.0]}"
                  },
                  {
                    "k": "location_list",
                    "v": "['WLG', 'AKL', 'DUD', 'CHC']"
                  },
                  {
                    "k": "vs30",
                    "v": "400"
                  },
                  {
                    "k": "disagg_conf",
                    "v": "{'enabled': False, 'config': {}}"
                  },
                  {
                    "k": "oq",
                    "v": "{'general': {'random_seed': 25, 'calculation_mode': 'classical', 'ps_grid_spacing': 30}, 'logic_tree': {'number_of_logic_tree_samples': 0}, 'erf': {'rupture_mesh_spacing': 4, 'width_of_mfd_bin': 0.1, 'complex_fault_mesh_spacing': 10.0, 'area_source_discretization': 10.0}, 'site_params': {'reference_vs30_type': 'measured'}, 'calculation': {'investigation_time': 1.0, 'truncation_level': 4, 'maximum_distance': {'Active Shallow Crust': '[[4.0, 0], [5.0, 100.0], [6.0, 200.0], [9.5, 300.0]]'}}, 'output': {'individual_curves': 'true'}}"
                  },
                  {
                    "k": "srm_logic_tree",
                    "v": "{'version': '', 'title': '', 'fault_systems': [{'short_name': 'HIK', 'long_name': 'Hikurangi-Kermadec', 'branches': [{'values': [{'name': 'dm', 'long_name': 'deformation model', 'value': 'TL'}, {'name': 'bN', 'long_name': 'bN pair', 'value': [1.097, 21.5]}, {'name': 'C', 'long_name': 'area-magnitude scaling', 'value': 4.0}, {'name': 's', 'long_name': 'moment rate scaling', 'value': 1.0}], 'sources': [{'nrml_id': 'SW52ZXJzaW9uU29sdXRpb25Ocm1sOjEyOTE2MDg=', 'rupture_rate_scaling': None, 'inversion_id': '', 'rupture_set_id': '', 'inversion_solution_type': '', 'type': 'inversion'}, {'nrml_id': 'RmlsZToxMzA3NDA=', 'rupture_rate_scaling': None, 'type': 'distributed'}], 'weight': 1.0, 'rupture_rate_scaling': 1.0}]}], 'logic_tree_version': 2}"
                  }
                ],
                "result": "SUCCESS",
                "hazard_solution": {
                  "id": "T3BlbnF1YWtlSGF6YXJkU29sdXRpb246NjcwMTI1Ng=="
                }
              }
            }
          },
          {
            "node": {
              "child": {
                "arguments": [
                  {
                    "k": "title",
                    "v": "OpenQuake Hazard Calcs"
                  },
                  {
                    "k": "description",
                    "v": "Logic Tree 9.0.1, locations for cave locations"
                  },
                  {
                    "k": "task_type",
                    "v": "HAZARD"
                  },
                  {
                    "k": "gmcm_logic_tree",
                    "v": "<?xml version=``1.0`` encoding=``UTF-8``?>--<nrml xmlns:gml=``http://www.opengis.net/gml``-      xmlns=``http://openquake.org/xmlns/nrml/0.4``>-    <logicTree logicTreeID='lt1'>-            <logicTreeBranchSet uncertaintyType=``gmpeModel`` branchSetID=``bs_crust``-                    applyToTectonicRegionType=``Active Shallow Crust``>--                    <logicTreeBranch branchID=``BSSA2014_center``>-                <uncertaintyModel>[BooreEtAl2014]-                  sigma_mu_epsilon = 0.0 </uncertaintyModel>-                        <uncertaintyWeight>1.0</uncertaintyWeight>-                    </logicTreeBranch>--            </logicTreeBranchSet>--            <logicTreeBranchSet uncertaintyType=``gmpeModel`` branchSetID=``bs_interface``-                    applyToTectonicRegionType=``Subduction Interface``>--                <logicTreeBranch branchID=``ATK22_SI_center``>-                    <uncertaintyModel>[Atkinson2022SInter]-                          epistemic = ``Central``-                            modified_sigma = ``true``-                           </uncertaintyModel>-                    <uncertaintyWeight>1.0</uncertaintyWeight>-                      </logicTreeBranch>--            </logicTreeBranchSet>--            <logicTreeBranchSet uncertaintyType=``gmpeModel`` branchSetID=``bs_slab``-                    applyToTectonicRegionType=``Subduction Intraslab``>--                <logicTreeBranch branchID=``ATK22_SS_center``>-                      <uncertaintyModel>[Atkinson2022SSlab]-                              epistemic = ``Central``-                              modified_sigma = ``true``-                              </uncertaintyModel>-                        <uncertaintyWeight>1.0</uncertaintyWeight>-                </logicTreeBranch>--            </logicTreeBranchSet>-    </logicTree>-</nrml>-"
                  },
                  {
                    "k": "model_type",
                    "v": "COMPOSITE"
                  },
                  {
                    "k": "intensity_spec",
                    "v": "{'tag': 'fixed', 'measures': ['PGA'], 'levels': [0.01, 0.02, 0.04, 0.06, 0.08, 0.1, 0.2, 0.3, 0.4, 0.5, 0.6, 0.7, 0.8, 0.9, 1.0, 1.2, 1.4, 1.6, 1.8, 2.0, 2.2, 2.4, 2.6, 2.8, 3.0, 3.5, 4.0, 4.5, 5.0]}"
                  },
                  {
                    "k": "location_list",
                    "v": "['WLG', 'AKL', 'DUD', 'CHC']"
                  },
                  {
                    "k": "vs30",
                    "v": "400"
                  },
                  {
                    "k": "disagg_conf",
                    "v": "{'enabled': False, 'config': {}}"
                  },
                  {
                    "k": "oq",
                    "v": "{'general': {'random_seed': 25, 'calculation_mode': 'classical', 'ps_grid_spacing': 30}, 'logic_tree': {'number_of_logic_tree_samples': 0}, 'erf': {'rupture_mesh_spacing': 4, 'width_of_mfd_bin': 0.1, 'complex_fault_mesh_spacing': 10.0, 'area_source_discretization': 10.0}, 'site_params': {'reference_vs30_type': 'measured'}, 'calculation': {'investigation_time': 1.0, 'truncation_level': 4, 'maximum_distance': {'Active Shallow Crust': '[[4.0, 0], [5.0, 100.0], [6.0, 200.0], [9.5, 300.0]]'}}, 'output': {'individual_curves': 'true'}}"
                  },
                  {
                    "k": "srm_logic_tree",
                    "v": "{'version': '', 'title': '', 'fault_systems': [{'short_name': 'CRU', 'long_name': 'Crustal', 'branches': [{'values': [{'name': 'dm', 'long_name': 'deformation model', 'value': 'geologic'}, {'name': 'td', 'long_name': 'time dependent', 'value': False}, {'name': 'bN', 'long_name': 'bN pair', 'value': [0.959, 3.4]}, {'name': 'C', 'long_name': 'area-magnitude scaling', 'value': 4.2}, {'name': 's', 'long_name': 'moment rate scaling', 'value': 1.0}], 'sources': [{'nrml_id': 'SW52ZXJzaW9uU29sdXRpb25Ocm1sOjEyOTE0NzY=', 'rupture_rate_scaling': None, 'inversion_id': '', 'rupture_set_id': '', 'inversion_solution_type': '', 'type': 'inversion'}, {'nrml_id': 'RmlsZToxMzA3MTk=', 'rupture_rate_scaling': None, 'type': 'distributed'}], 'weight': 1.0, 'rupture_rate_scaling': 1.0}]}], 'logic_tree_version': 2}"
                  }
                ],
                "result": "SUCCESS",
                "hazard_solution": {
                  "id": "T3BlbnF1YWtlSGF6YXJkU29sdXRpb246NjcwMTI1OA=="
                }
              }
            }
          },
          {
            "node": {
              "child": {
                "arguments": [
                  {
                    "k": "title",
                    "v": "OpenQuake Hazard Calcs"
                  },
                  {
                    "k": "description",
                    "v": "Logic Tree 9.0.1, locations for cave locations"
                  },
                  {
                    "k": "task_type",
                    "v": "HAZARD"
                  },
                  {
                    "k": "gmcm_logic_tree",
                    "v": "<?xml version=``1.0`` encoding=``UTF-8``?>--<nrml xmlns:gml=``http://www.opengis.net/gml``-      xmlns=``http://openquake.org/xmlns/nrml/0.4``>-    <logicTree logicTreeID='lt1'>-            <logicTreeBranchSet uncertaintyType=``gmpeModel`` branchSetID=``bs_crust``-                    applyToTectonicRegionType=``Active Shallow Crust``>--                    <logicTreeBranch branchID=``BSSA2014_center``>-                <uncertaintyModel>[BooreEtAl2014]-                  sigma_mu_epsilon = 0.0 </uncertaintyModel>-                        <uncertaintyWeight>1.0</uncertaintyWeight>-                    </logicTreeBranch>--            </logicTreeBranchSet>--            <logicTreeBranchSet uncertaintyType=``gmpeModel`` branchSetID=``bs_interface``-                    applyToTectonicRegionType=``Subduction Interface``>--                <logicTreeBranch branchID=``ATK22_SI_center``>-                    <uncertaintyModel>[Atkinson2022SInter]-                          epistemic = ``Central``-                            modified_sigma = ``true``-                           </uncertaintyModel>-                    <uncertaintyWeight>1.0</uncertaintyWeight>-                      </logicTreeBranch>--            </logicTreeBranchSet>--            <logicTreeBranchSet uncertaintyType=``gmpeModel`` branchSetID=``bs_slab``-                    applyToTectonicRegionType=``Subduction Intraslab``>--                <logicTreeBranch branchID=``ATK22_SS_center``>-                      <uncertaintyModel>[Atkinson2022SSlab]-                              epistemic = ``Central``-                              modified_sigma = ``true``-                              </uncertaintyModel>-                        <uncertaintyWeight>1.0</uncertaintyWeight>-                </logicTreeBranch>--            </logicTreeBranchSet>-    </logicTree>-</nrml>-"
                  },
                  {
                    "k": "model_type",
                    "v": "COMPOSITE"
                  },
                  {
                    "k": "intensity_spec",
                    "v": "{'tag': 'fixed', 'measures': ['PGA'], 'levels': [0.01, 0.02, 0.04, 0.06, 0.08, 0.1, 0.2, 0.3, 0.4, 0.5, 0.6, 0.7, 0.8, 0.9, 1.0, 1.2, 1.4, 1.6, 1.8, 2.0, 2.2, 2.4, 2.6, 2.8, 3.0, 3.5, 4.0, 4.5, 5.0]}"
                  },
                  {
                    "k": "location_list",
                    "v": "['WLG', 'AKL', 'DUD', 'CHC']"
                  },
                  {
                    "k": "vs30",
                    "v": "400"
                  },
                  {
                    "k": "disagg_conf",
                    "v": "{'enabled': False, 'config': {}}"
                  },
                  {
                    "k": "oq",
                    "v": "{'general': {'random_seed': 25, 'calculation_mode': 'classical', 'ps_grid_spacing': 30}, 'logic_tree': {'number_of_logic_tree_samples': 0}, 'erf': {'rupture_mesh_spacing': 4, 'width_of_mfd_bin': 0.1, 'complex_fault_mesh_spacing': 10.0, 'area_source_discretization': 10.0}, 'site_params': {'reference_vs30_type': 'measured'}, 'calculation': {'investigation_time': 1.0, 'truncation_level': 4, 'maximum_distance': {'Active Shallow Crust': '[[4.0, 0], [5.0, 100.0], [6.0, 200.0], [9.5, 300.0]]'}}, 'output': {'individual_curves': 'true'}}"
                  },
                  {
                    "k": "srm_logic_tree",
                    "v": "{'version': '', 'title': '', 'fault_systems': [{'short_name': 'SLAB', 'long_name': 'Intra-slab', 'branches': [{'values': [{'name': 'r', 'long_name': 'earthquake rates', 'value': 'uniform'}, {'name': 'd', 'long_name': 'hypocentral depths', 'value': 1}], 'sources': [{'nrml_id': 'RmlsZToxMjEwMzM=', 'rupture_rate_scaling': None, 'type': 'distributed'}], 'weight': 1.0, 'rupture_rate_scaling': 1.0}]}], 'logic_tree_version': 2}"
                  }
                ],
                "result": "SUCCESS",
                "hazard_solution": {
                  "id": "T3BlbnF1YWtlSGF6YXJkU29sdXRpb246NjcwMTI2MA=="
                }
              }
            }
          }
        ]
      }
    }
  }
}