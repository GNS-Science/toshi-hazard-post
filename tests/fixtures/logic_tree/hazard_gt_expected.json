{"SW52ZXJzaW9uU29sdXRpb25Ocm1sOjEyMDg3NA==:RmlsZToxMzA3MTM=": "T3BlbnF1YWtlSGF6YXJkU29sdXRpb246MTI5NDg0", "SW52ZXJzaW9uU29sdXRpb25Ocm1sOjEyMDk1NQ==:RmlsZToxMzA3Mjg=": "T3BlbnF1YWtlSGF6YXJkU29sdXRpb246MTI5NDUx", "SW52ZXJzaW9uU29sdXRpb25Ocm1sOjExODcyMw==:RmlsZToxMzA3NTQ=": "T3BlbnF1YWtlSGF6YXJkU29sdXRpb246MTI5MzQ0", "SW52ZXJzaW9uU29sdXRpb25Ocm1sOjEyMDkxMQ==:RmlsZToxMzA3MTA=": "T3BlbnF1YWtlSGF6YXJkU29sdXRpb246MTI5NDcy", "SW52ZXJzaW9uU29sdXRpb25Ocm1sOjEyMDg4MQ==:RmlsZToxMzA3MjU=": "T3BlbnF1YWtlSGF6YXJkU29sdXRpb246MTI5NDY5", "SW52ZXJzaW9uU29sdXRpb25Ocm1sOjExODcyOQ==:RmlsZToxMzA3NTU=": "T3BlbnF1YWtlSGF6YXJkU29sdXRpb246MTI5MzQy", "SW52ZXJzaW9uU29sdXRpb25Ocm1sOjExNDQwOQ==:RmlsZToxMzA3NDg=": "T3BlbnF1YWtlSGF6YXJkU29sdXRpb246MTI5Mzgz", "SW52ZXJzaW9uU29sdXRpb25Ocm1sOjEyMDkxMg==:RmlsZToxMzA3Mjg=": "T3BlbnF1YWtlSGF6YXJkU29sdXRpb246MTI5NDUw", "SW52ZXJzaW9uU29sdXRpb25Ocm1sOjEyMDk2Mw==:RmlsZToxMzA3MTA=": "T3BlbnF1YWtlSGF6YXJkU29sdXRpb246MTI5NDky", "SW52ZXJzaW9uU29sdXRpb25Ocm1sOjEyMDk1Mw==:RmlsZToxMzA3MTY=": "T3BlbnF1YWtlSGF6YXJkU29sdXRpb246MTI5NDQ5", "SW52ZXJzaW9uU29sdXRpb25Ocm1sOjEyMDkwNA==:RmlsZToxMzA3MTM=": "T3BlbnF1YWtlSGF6YXJkU29sdXRpb246MTI5NDc0", "SW52ZXJzaW9uU29sdXRpb25Ocm1sOjEyMDg4OQ==:RmlsZToxMzA3MTk=": "T3BlbnF1YWtlSGF6YXJkU29sdXRpb246MTI5NDU4", "SW52ZXJzaW9uU29sdXRpb25Ocm1sOjEyMDg5Nw==:RmlsZToxMzA3MTA=": "T3BlbnF1YWtlSGF6YXJkU29sdXRpb246MTI5NDg5", "SW52ZXJzaW9uU29sdXRpb25Ocm1sOjEyMDg5Mw==:RmlsZToxMzA3MTY=": "T3BlbnF1YWtlSGF6YXJkU29sdXRpb246MTI5NDc2", "SW52ZXJzaW9uU29sdXRpb25Ocm1sOjEyMDk1NA==:RmlsZToxMzA3MjI=": "T3BlbnF1YWtlSGF6YXJkU29sdXRpb246MTI5NDY0", "SW52ZXJzaW9uU29sdXRpb25Ocm1sOjExODcxNw==:RmlsZToxMzA3NTM=": "T3BlbnF1YWtlSGF6YXJkU29sdXRpb246MTI5MzQz", "SW52ZXJzaW9uU29sdXRpb25Ocm1sOjExNDQwNg==:RmlsZToxMzA3NDY=": "T3BlbnF1YWtlSGF6YXJkU29sdXRpb246MTI5Mzgy", "SW52ZXJzaW9uU29sdXRpb25Ocm1sOjExNDQyOQ==:RmlsZToxMzA3MzY=": "T3BlbnF1YWtlSGF6YXJkU29sdXRpb246MTI5Mzk1", "SW52ZXJzaW9uU29sdXRpb25Ocm1sOjEyMDkxNw==:RmlsZToxMzA3MTY=": "T3BlbnF1YWtlSGF6YXJkU29sdXRpb246MTI5NDYz", "SW52ZXJzaW9uU29sdXRpb25Ocm1sOjExNDQyOA==:RmlsZToxMzA3MzI=": "T3BlbnF1YWtlSGF6YXJkU29sdXRpb246MTI5Mzk2", "SW52ZXJzaW9uU29sdXRpb25Ocm1sOjEyMDk2Nw==:RmlsZToxMzA3MTk=": "T3BlbnF1YWtlSGF6YXJkU29sdXRpb246MTI5NDgz", "SW52ZXJzaW9uU29sdXRpb25Ocm1sOjEyMDkwOA==:RmlsZToxMzA3MjI=": "T3BlbnF1YWtlSGF6YXJkU29sdXRpb246MTI5NDY2", "SW52ZXJzaW9uU29sdXRpb25Ocm1sOjEyMDk3Mw==:RmlsZToxMzA3MjU=": "T3BlbnF1YWtlSGF6YXJkU29sdXRpb246MTI5NDc1", "SW52ZXJzaW9uU29sdXRpb25Ocm1sOjEyMDk2NQ==:RmlsZToxMzA3MDc=": "T3BlbnF1YWtlSGF6YXJkU29sdXRpb246MTI5NDcw", "SW52ZXJzaW9uU29sdXRpb25Ocm1sOjEyMDkwNg==:RmlsZToxMzA3MTk=": "T3BlbnF1YWtlSGF6YXJkU29sdXRpb246MTI5NDkz", "SW52ZXJzaW9uU29sdXRpb25Ocm1sOjEyMDkxMw==:RmlsZToxMzA3MjU=": "T3BlbnF1YWtlSGF6YXJkU29sdXRpb246MTI5NDYx", "SW52ZXJzaW9uU29sdXRpb25Ocm1sOjEyMDg5Mg==:RmlsZToxMzA3Mjg=": "T3BlbnF1YWtlSGF6YXJkU29sdXRpb246MTI5NDcx", "SW52ZXJzaW9uU29sdXRpb25Ocm1sOjEyMDg4Ng==:RmlsZToxMzA3MDc=": "T3BlbnF1YWtlSGF6YXJkU29sdXRpb246MTI5NDc4", "SW52ZXJzaW9uU29sdXRpb25Ocm1sOjExNDQxNg==:RmlsZToxMzA3NDI=": "T3BlbnF1YWtlSGF6YXJkU29sdXRpb246MTI5Mzg3", "SW52ZXJzaW9uU29sdXRpb25Ocm1sOjEyMDkyNA==:RmlsZToxMzA3MzE=": "T3BlbnF1YWtlSGF6YXJkU29sdXRpb246MTI5NDYy", "SW52ZXJzaW9uU29sdXRpb25Ocm1sOjExNDQxMw==:RmlsZToxMzA3NDA=": "T3BlbnF1YWtlSGF6YXJkU29sdXRpb246MTI5Mzg1", "SW52ZXJzaW9uU29sdXRpb25Ocm1sOjExNDQyNA==:RmlsZToxMzA3Mzg=": "T3BlbnF1YWtlSGF6YXJkU29sdXRpb246MTI5Mzg2", "SW52ZXJzaW9uU29sdXRpb25Ocm1sOjEyMDg4Mg==:RmlsZToxMzA3MzE=": "T3BlbnF1YWtlSGF6YXJkU29sdXRpb246MTI5NDY3", "SW52ZXJzaW9uU29sdXRpb25Ocm1sOjEyMDk1Ng==:RmlsZToxMzA3MTM=": "T3BlbnF1YWtlSGF6YXJkU29sdXRpb246MTI5NDY1", "SW52ZXJzaW9uU29sdXRpb25Ocm1sOjEyMDg5OA==:RmlsZToxMzA3MDc=": "T3BlbnF1YWtlSGF6YXJkU29sdXRpb246MTI5NDY4", "SW52ZXJzaW9uU29sdXRpb25Ocm1sOjEyMDg3MQ==:RmlsZToxMzA3MjI=": "T3BlbnF1YWtlSGF6YXJkU29sdXRpb246MTI5NDcz", "SW52ZXJzaW9uU29sdXRpb25Ocm1sOjExNDQxNw==:RmlsZToxMzA3MzQ=": "T3BlbnF1YWtlSGF6YXJkU29sdXRpb246MTI5Mzk0", "SW52ZXJzaW9uU29sdXRpb25Ocm1sOjExNDQxNA==:RmlsZToxMzA3NDQ=": "T3BlbnF1YWtlSGF6YXJkU29sdXRpb246MTI5Mzg0", ":RmlsZToxMjEwMzM=": "T3BlbnF1YWtlSGF6YXJkU29sdXRpb246MTI5NDE4", "SW52ZXJzaW9uU29sdXRpb25Ocm1sOjEyMDk0NA==:RmlsZToxMzA3MTk=": "T3BlbnF1YWtlSGF6YXJkU29sdXRpb246MTI5NDc3", "SW52ZXJzaW9uU29sdXRpb25Ocm1sOjEyMDkyNw==:RmlsZToxMzA3MTA=": "T3BlbnF1YWtlSGF6YXJkU29sdXRpb246MTI5NDQ4", "SW52ZXJzaW9uU29sdXRpb25Ocm1sOjEyMDkzMQ==:RmlsZToxMzA3Mjg=": "T3BlbnF1YWtlSGF6YXJkU29sdXRpb246MTI5NDk5", "SW52ZXJzaW9uU29sdXRpb25Ocm1sOjEyMDk3NA==:RmlsZToxMzA3MzE=": "T3BlbnF1YWtlSGF6YXJkU29sdXRpb246MTI5NDk1", "SW52ZXJzaW9uU29sdXRpb25Ocm1sOjEyMDkzMg==:RmlsZToxMzA3MTM=": "T3BlbnF1YWtlSGF6YXJkU29sdXRpb246MTI5NTAy", "SW52ZXJzaW9uU29sdXRpb25Ocm1sOjEyMDk1MQ==:RmlsZToxMzA3MTY=": "T3BlbnF1YWtlSGF6YXJkU29sdXRpb246MTI5NDk3", "SW52ZXJzaW9uU29sdXRpb25Ocm1sOjEyMDk0OA==:RmlsZToxMzA3MjI=": "T3BlbnF1YWtlSGF6YXJkU29sdXRpb246MTI5NDk0", "SW52ZXJzaW9uU29sdXRpb25Ocm1sOjEyMDk0Mw==:RmlsZToxMzA3MzE=": "T3BlbnF1YWtlSGF6YXJkU29sdXRpb246MTI5NDk4", "SW52ZXJzaW9uU29sdXRpb25Ocm1sOjEyMDkzNg==:RmlsZToxMzA3MDc=": "T3BlbnF1YWtlSGF6YXJkU29sdXRpb246MTI5NTAw", "SW52ZXJzaW9uU29sdXRpb25Ocm1sOjEyMDkzNA==:RmlsZToxMzA3MjU=": "T3BlbnF1YWtlSGF6YXJkU29sdXRpb246MTI5NDk2"}